Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -298,7 +298,8 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
     auto transientZoomOrigin = std::exchange(m_transientZoomOrigin, { });
 
     auto rootScrollingNodeID = m_webPageProxy->scrollingCoordinatorProxy()->rootScrollingNodeID();
-    m_webPageProxy->scrollingCoordinatorProxy()->deferWheelEventTestCompletionForReason(rootScrollingNodeID, WheelEventTestMonitorDeferReason::CommittingTransientZoom);
+    if (m_webPageProxy->scrollingCoordinatorProxy()->isMonitoringWheelEvents())
```

Comment:
Maybe it would be better if this check was moved into the function that sends the IPC? In case any more calls to this are added in the future.

Reply:
I originally wanted to avoid double checking but its probably fine as it would prevent this problem in the future.